sonar-csharp-plugin/pom.xml

```diff
@@ -260,7 +260,7 @@
                   </fileset>
                 </copy>
                 <exec executable="dotnet"
-                      dir="${analyzers.directory}/src/SonarAnalyzer.RuleDescriptorGenerator/bin/Release/net6">
+                      dir="${analyzers.directory}/packaging/binaries/RuleDescriptorGenerator">
```

Comment:
Can you use `@(BinariesToCopy)` somehow here?

Reply:
No.
We're in maven, it has no clue what MsBuild does.
We're in a different stage (`Build` vs `Tests`) so those binaries even do not exist on that original path because we have a different VM instance.